appveyor.yml

```diff
@@ -0,0 +1,236 @@
+# As config was originally based on an example by Olivier Grisel. Thanks!
+# https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
+
+clone_depth: 50
+
+# No reason for us to restrict the number concurrent jobs
+max_jobs: 100
+
+environment:
+  global:
+      MINGW_32: C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin
+      MINGW_64: C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin
+      OPENBLAS_32: https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.2.19_win32.zip
+      OPENBLAS_64: https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.2.19_win64.zip
+      NUMPY_HEAD: https://github.com/xoviat/numpy.git
```

Comment:
So the best way that I could come with is to swap out the "distutils" folder with that from numpy HEAD. This allows build failures as a result of numpy.distutils to be repaired rapidly rather than waiting on a numpy release, and it is still possible to build against old numpy versions.

The link will be updated after the numpy PR is merged.